### PR TITLE
[Product Partnerships] Added Timestream Quickstart 

### PR DESCRIPTION
**Pull request by rahul188**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

**Review by bitcoinfinancier — APPROVED** [review]

**Comment by aswanson-nr** [comment]
Hey @rahul188, I'm going to toss this into the "draft" state until it's ready to merge!

**Comment by rahul188** [comment]
@aswanson-nr, Can you please merge? I have removed the **Do Not Merge** Tag.

File: quickstarts/aws/amazon-timestream/config.yml
Comment (seemantk) [inline quickstarts/aws/amazon-timestream/config.yml:4]:
```suggestion
  Amazon Timestream is a serverless scalable time-series database service to store and analyze up to trillions of events per day. It automatically scales up or down to adjust capacity and performance.
```

The "1000 times faster" seems a bit marketingy for AWS (faster than what, for example?).  You might consider adjusting the copy to be more of a neutral description of Amz Timestream

File: quickstarts/aws/amazon-timestream/config.yml
Comment (seemantk) [inline quickstarts/aws/amazon-timestream/config.yml:6]:
```suggestion
  Start monitoring Amazon Timestream by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
```

**Review by seemantk — COMMENTED** [review]
Looks great. Only some small changes requested

File: quickstarts/aws/amazon-timestream/config.yml
Comment (rahul188) [reply quickstarts/aws/amazon-timestream/config.yml]: @seemantk, addressed, I have updated the description.

File: quickstarts/aws/amazon-timestream/config.yml
Comment (rahul188) [reply quickstarts/aws/amazon-timestream/config.yml]: @seemantk, addressed, I have updated the description.

**Review by brammerl — APPROVED** [review]

**Pull request merged**
